File: Resultados/combine.py
import numpy as np
import csv


def main():

	names_array = ['iris', 'wine', 'soybean', 'pima', 'balance', 'boston', 'diabetes', 
	'breast_cancer', 'bupa', 'ecoli', 'haberman', 'led7digit', 'monk2', 'newthyroid', 
	'vehicle', 'zoo', 'sonar', 'heart', 'ionosphere', 'wdbc', 'vowel', 'movement_libras', 
	'appendicitis', 'saheart', 'spectfheart', 'hayesroth', 'tae', 'glass', 'rand', 'spiral', 'moons', 'circles', "means"]

	data = np.loadtxt("tabla10.dat")
	
	for i in range(data.shape[0]):

		print(names_array[i].title(), end='')

		for j in range(data.shape[1]):

			if j != 3:
				print(" & %.3f" % (data[i, j]), end='')
			else:
				print(" && %.3f" % (data[i, j]), end='')

		print(" \\\\")


if __name__ == "__main__": main()

File: Resultados/combine.r

tabla5.1 <- read.csv("SHADEvsBRKGA/tabla5.csv", dec = ",", row.names = 1)
tabla5.2 <- read.csv("SHADEvsBRKGA-2/tabla5.csv", dec = ",", row.names = 1)

tabla10.1 <- read.csv("SHADEvsBRKGA/tabla10.csv", dec = ",", row.names = 1)
tabla10.2 <- read.csv("SHADEvsBRKGA-2/tabla10.csv", dec = ",", row.names = 1)

tabla15.1 <- read.csv("SHADEvsBRKGA/tabla15.csv", dec = ",", row.names = 1)
tabla15.2 <- read.csv("SHADEvsBRKGA-2/tabla15.csv", dec = ",", row.names = 1)

tabla20.1 <- read.csv("SHADEvsBRKGA/tabla20.csv", dec = ",", row.names = 1)
tabla20.2 <- read.csv("SHADEvsBRKGA-2/tabla20.csv", dec = ",", row.names = 1)

tabla5 = tabla5.1 * 0.4 + tabla5.2 * 0.6
tabla10 = tabla10.1 * 0.4 + tabla10.2 * 0.6
tabla15 = tabla15.1 * 0.4 + tabla15.2 * 0.6
tabla20 = tabla20.1 * 0.4 + tabla20.2 * 0.6

tabla5 = tabla5[!(row.names(tabla5) %in% 
                     c("Wine ", "Balance ", "Boston ", "Diabetes ", "Newthyroid ", "Heart ", "Rand ")), ]

tabla10 = tabla10[!(row.names(tabla10) %in% 
                      c("Wine ", "Balance ", "Boston ", "Diabetes ", "Newthyroid ", "Heart ", "Rand ")), ]

tabla15 = tabla15[!(row.names(tabla15) %in% 
                      c("Wine ", "Balance ", "Boston ", "Diabetes ", "Newthyroid ", "Heart ", "Rand ")), ]

tabla20 = tabla20[!(row.names(tabla20) %in% 
                      c("Wine ", "Balance ", "Boston ", "Diabetes ", "Newthyroid ", "Heart ", "Rand ")), ]

write.table(tabla5, "tabla5.dat", row.names = F, col.names = F)
write.table(tabla10, "tabla10.dat", row.names = F, col.names = F)
write.table(tabla15, "tabla15.dat", row.names = F, col.names = F)
write.table(tabla20, "tabla20.dat", row.names = F, col.names = F)
